pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -160,6 +160,12 @@
                  help=u'Show help message for specifying INSTANCENAME '
                       u'including use of the --key and --namespace options.')]
 
+detail_option = [              # pylint: disable=invalid-name
+    click.option('--detail', 'detail', is_flag=True, required=False,
+                 help=u'In the TABLE output formats, show properties with no '
+                      u'value in any instances to be displayed. Otherwise only '
+                      u'properties value at least one instance are displayed')]
+
```

Comment:
I struggle with the option name `--detail`. It does not really show more details about the items it shows, it shows more items (properties). I think `--include-null-properties` or `--show-null-properties` would appropriately describe what it does, but it is a bit long. How about `--show-null`?

Reply:
I have no issue with this change and agree that --show-null works.  Making change now

Reply:
DONE